src/components/Project/AragonNetwork.js

```diff
@@ -0,0 +1,203 @@
+                representatives) who can vote on their behalf and continue
+                experimenting with new voting mechanisms.
+              </h3>
+            </Module>
+          </Entry>
+          <Entry>
+            <Module>
+              <div className="badge-container">
+                <Badge>Dispute Resolution</Badge>
+                <AragonOneBadge>Aragon One</AragonOneBadge>
+              </div>
+              <h1>Staking and Lock Managers</h1>
+              <h3>
+                Provides a secure foundation for managing collateral deposits
+                for agreements while allowing collateral assets to be used in to
+                participate in governance.
+              </h3>
+            </Module>
+          </Entry>
+          <Entry>
+            <Module>
+              <div className="badge-container">
+                <Badge>Dispute Resolution</Badge>
+                <AragonOneBadge>Aragon One</AragonOneBadge>
+              </div>
+              <h1>Aragon Court PoC</h1>
+              <h3>
+                This deliverable includes the Aragon Court protocol, an
+                interface for users of the court to review and manage agreements
+                and disputes, and an interface for jurors to participate in the
+                arbitration process.
+              </h3>
+            </Module>
+          </Entry>
+          <Entry entrypoint>
+            <Title>Near-term</Title>
+            <Subtitle>
+              Wider areas of focus with some flexibility on the scope.
+            </Subtitle>
+          </Entry>
+          <Entry>
+            <Module>
+              <div className="badge-container">
+                <Badge>Dispute Resolution</Badge>
+                <AragonOneBadge>Aragon One</AragonOneBadge>
+              </div>
+              <h1>Proposals agreements</h1>
+              <h3>
+                Aragon Agreements are a core component of the Aragon Network,
+                they enable users to define human-readable obligations and lock
+                collateral to provide assurances to their counter-parties.
+              </h3>
+            </Module>
+          </Entry>
+          <Entry>
+            <Module>
+              <div className="badge-container">
+                <EnhancedBadge>Enhanced Voting</EnhancedBadge>
+                <AragonOneBadge>Aragon One</AragonOneBadge>
+              </div>
+              <h1>Vote Relay Protocol</h1>
+              <h3>
+                Improve the experience and scalability of voting by implementing
+                a protocol for votes to be submitted to a set of bonded
+                relayers.
+              </h3>
+            </Module>
+          </Entry>
+          <Entry lastpoint />
+        </div>
+      </RoadmapContainer>
+    </Container>
+  </RoadmapSection>
+);
+
+const EnhancedBadge = styled(Badge)`
+  background: #d5e8ff80;
+  color: #4188dc;
+`;
+
+const RoadmapContainer = styled.div`
```

Comment:
I think you could fit all these styles into the `RoadMapContainer` no? or is there a specific reason why you decided not to use a `styled-component` for the `<div classname="roadmap-container">` (you could call it `Wrap`).

If you use a wrapping component you can also use `&` for `:after` and `:before`:

```JavaScript
const Wrap = styled.div`
  width: 100%;
  max-width: 870px;
  padding: 0;
  ${medium('padding: 5px 50px;')};
  position: relative;
  margin: auto;

  &:before { /*...*/ }
  &:after { /*...*/ }
`
```